next/components/forms/SingleCheckBox.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
```

Comment:
do we need both singlecheckbox and checkboxgroupitem ?

if there are differences in functionality which are hard to reconcile (and which warrant two separate components) they should at least be backed up by same visual component, since they look the same in the end.

But a completely ideal way would be for SIngleCheckBox and CHeckBoxGroup to work in such way that the former can be placed in the latter and the 'api' works and makes sense.